_docs/api/file-object.md

```diff
@@ -37,14 +37,15 @@ redirect_from:
     - /docs/v1.0.1/api/file-object/
     - /docs/v1.1.0/api/file-object/
     - /docs/v1.1.1/api/file-object/
+    - /docs/v1.1.2/api/file-object/
     - /docs/latest/api/file-object/
 source_url: 'https://github.com/electron/electron/blob/master/docs/api/file-object.md'
 excerpt: "Use the HTML5 <code>File</code> API to work natively with files on the filesystem."
-title: "File object"
+title: "File Object"
```

Comment:
I think https://github.com/electron/electron.atom.io/blob/93f846bc69bfe1b2d21bfaad3baddb6e403bc64d/_data/processes.yml#L12 needs to be updated since this title is changing.